src/main/java/blackjack/domain/card/Denomination.java

```diff
@@ -0,0 +1,48 @@
+package blackjack.domain.card;
+
+import java.util.Arrays;
+
+public enum Denomination {
+    ACE("A", 1),
+    TWO("2", 2),
+    THREE("3", 3),
+    FOUR("4", 4),
+    FIVE("5", 5),
+    SIX("6", 6),
+    SEVEN("7", 7),
+    EIGHT("8", 8),
+    NINE("9", 9),
+    TEN("10", 10),
+    JACK("J", 10),
+    QUEEN("Q", 10),
+    KING("K", 10);
+
+    private static final String ACE_NAME = "A";
+
+    private String name;
+    private int score;
+
+    Denomination(String name, int score) {
+        this.name = name;
+        this.score = score;
+    }
+
+    public static Denomination of(String value) {
+        return Arrays.stream(Denomination.values())
+            .filter(denomination -> denomination.getName().equals(value))
+            .findFirst()
```

Comment:
```suggestion
        return Arrays.stream(Denomination.values())
            .filter(denomination -> denomination.name.equals(value))
            .findFirst()
```

getter를 사용하지 않는 방법도 있습니다 :)